### PR TITLE
(maint) Remove long dead/unused tasks and code

### PR DESCRIPTION
**Pull request by haus**
This commit collects a few of the tasks that are no longer in active use
at Puppet Labs and removes them. The remote_build and pe_remote tasks
have been superceded by the pl:jenkins:uber_build task and the release
tasks have been long since replaced by the pl:jenkins:uber_ship task.
They are no longer loaded in rake_utils and mention of them is removed
from the README.

### CODE REVIEW

**Comment by underscorgan** [comment]
@haus I don't hate this idea but worry that things might explode.

**Comment by haus** [comment]
@underscorgan Yea, I haven't tested this at all, so it definitely needs to be tested in some fashion.
Having said that...these tasks aren't used or called by anything currently in use, so I'm pretty confident this won't break stuff.

**Comment by underscorgan** [comment]
I'd still like to see some testing, but since grep isn't turning anything up I'm :+1:

**Comment by melissa** [comment]
I want this. But I'm so scared of this.

**Comment by haus** [comment]
@underscorgan i can walk you through the tests i would do to make sure this isn't going to break the world.

File: README.md [hunk 1]
```diff
@@ -921,50 +921,6 @@ files:
...
-    Perform `pl:deb_all` on the remote debian build host as specified in the
-    team build_data.yaml file
-
-* **pl:remote:dmg**
-
-
-    Perform `package:apple` on the remote Mac build host as specified in the
-    team build_data.yaml file
-
 * **pl:remote:freight**
```
Comment (melissa) [inline README.md]: isn't this `pl:remote:update_apt_repo`?

Comment (haus) [inline README.md]: pl:remote:freight still exists and now aliases to `update_apt_repo`. These docs are very stale.

**Comment by underscorgan** [comment]
This was tested on facter 2.x, puppetdb 2.3.x, and pe-razor-server 2015.2.x, so :+1:

**Comment by haus** [comment]
also pe-puppet 3.8.x

**Comment by underscorgan** [comment]
Closing since #464 was merged

**Pull request closed without merging**